workspace.json

```diff
@@ -6493,17 +6487,6 @@
             "cwd": "apps/services/party-letter-registry-api"
           }
         },
-        "schemas/build-openapi": {
-          "builder": "@nrwl/workspace:run-commands",
-          "options": {
-            "outputPath": "apps/services/party-letter-registry-api/src/openapi.yaml",
-            "commands": [
-              "nx run services-party-letter-registry-api:build",
-              "node dist/apps/services/party-letter-registry-api/buildOpenApi.js"
-            ],
-            "parallel": false
-          }
-        },
```

Comment:
Will this not break all consumers of this service?

Reply:
This service currently has no consumers. We are currently integrating this service [here](https://github.com/island-is/island.is/pull/3941).
This service is also not currently deployed to dev.
I opted for not altering the service in this PR to avoid conflicts with the integration PR.
We'll fix the client generation in that PR